cs/configuring.texy

```diff
@@ -345,3 +344,17 @@ Například takto může vypadat definice služby pojmenované `database`, což
 services:
 	database: PDO('mysql:host=127.0.0.1;dbname=test', root, password)
 \--
+
+
+Rozšíření
+=========
+
+Pomocí rozšíření můžeme rozdělit naši aplikaci do malých nezavislých častí s vlastní konfigurací. Nette obsahuje řadu předdefinovaných rozšíření, o kterých se dočtete více v kapitole [DI: Základní rozšíření pro kontejner |di-defaultextensions].
```

Comment:
Jelikož vlastně celá tato stránka je čistě o rozšířeních, neuváděl bych `rozšíření` jako něco dalšího. Možná třeba „Další rozšíření“

Určitě neumí aplikaci rozdělit do malých nezávislých částí :-) A Nette obsahuje jen několik, ne řadu rozšíření.

Spěchám teď na vlak, tak nestíhám to vymyslet jinak, ale chtělo by to říct, že kromě těchto uvedených sekcí je možné přidávat další, říká se jim rozšíření, několik jich obsahuje samotné nette a ukázat jak lze přidat třeba to dibi. 

Dokonce bych možná decorator, php a constants zmínil jmenovitě přímo v této stránce (s odkazem), protože jsou to sekce, na které člověk v configu narazí.